src/Movie-Card/Movie-Card.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import './Movie-Card.css'
+
+const MovieCard = ({ id, title, poster }) => {
+    return <img key={id} src={poster} id={id} name={title} alt={title} className='movie-card' height='300' width='200'/>
+}
+
+export default MovieCard;
```

Comment:
is the height and width being defined here ok or do you prefer to do all of that in styling?